objective.py

```diff
@@ -21,6 +21,9 @@ def set_data(self, X, y):
         self.lmbd = self.reg * self._get_lambda_max()
         self.n_features = self.X.shape[1]
 
+    def get_one_solution(self):
+        return np.zeros(self.n_features)
```

Comment:
doesn't this need to be different if self.fit_intercept is True ?

Reply:
Indeed, good catch.